USER_MANUAL.md

```diff
@@ -55,5 +55,25 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
   2. Additionally a handover value is incremented to monitor how many times this is called.
   3. The stack count, and the handover counter are then checked they are the same.
 
+#### QA Workflows Testing
```

Comment:
Do we want to have a completely different section with different gyms? Can these gyms not be described the same way as the other ones and then workflow testing just says to look at those gyms?

Reply:
I can totally do that, I thought it might be better to keep separate, but happy to just include them without separate header, as I've already got reference to them in the test cases